Add initial Terms of Use content. (#47)
wso2/choreo-docs: privacy-and-terms/terms-of-use.md

## 1. Use of the Services

**1.1.** To use Choreo, You must register for the Service and have a valid account. You may access and use the Service only in accordance with these Terms, all applicable laws  and, if You are using a paid tier, any subscription limits associated with your purchase. Subject to the above, WSO2 grants You and your Users a limited, non-exclusive, royalty free, non-transferable and terminable license to access and use Choreo solely for Your operations. No rights with respect to Choreo are granted to you other than those specifically detailed in these Terms.

**1.2.** Updates and upgrades. WSO2 provides the Service to you and may upgrade the functionality and user interface of the Service from time to time in its sole discretion and in accordance with these terms. These Terms shall also apply to any updates, upgrades, modules or features subsequently provided by WSO2 to You. You acknowledge that such upgrades may result in changes to the appearance and/or functionality of the Service.

> You agree that your purchases here under are neither contingent on the delivery of any future functionality or features nor dependent on any oral or written public comments made by us regarding future functionality or features.

Reviewer: ```suggestion
You agree that your purchases hereunder are neither contingent on the delivery of any future functionality or features nor dependent on any oral or written public comments made by us regarding future functionality or features.
```

Author: Changed as suggested.

> **1.3.** **Customer support:** WSO2 provides online and email technical support through the Help/Contact Support menu in the Choreo Developer Portal. If for any reason this menu is not available, you may contact WSO2 support by e-mail at the following address: help@choreo.io  In order for WSO2 to process your support request, you shall: (a) identify yourself (b) describe the nature of the support request; and (c) provide contact information in the form of email for follow-up communications.<br/><br/>

Reviewer: ```suggestion
**1.3.** **Customer support:** WSO2 provides online and email technical support through the Help/Contact Support menu in the Choreo Developer Portal. If for any reason this menu is not available, you may contact WSO2 support by e-mail at the following address: help@choreo.dev  In order for WSO2 to process your support request, you shall: (a) identify yourself (b) describe the nature of the support request; and (c) provide contact information in the form of email for follow-up communications.<br/><br/>
```

Author: Changed as suggested.

## 9. Release

> If You have a dispute with one or more users, You hereby release WSO2, its subsidiaries, affiliates, partners, agents, officers and employees from claims, demands and damages (direct and consequential) of every kind and nature, known and unknown arising out of or in any way connected with such disputes. If You are a California resident or an entity incorporated in California, You waive the applicability of California Civil Code Section 1542 which says A general release does not extend to claims which the creditor does not know or suspect to exist in such creditor’s favour at the time of executing the release, which if known by the creditor must have materially affected such credtor’s settlement with the debtor

Reviewer: ```suggestion
If You have a dispute with one or more users, You hereby release WSO2, its subsidiaries, affiliates, partners, agents, officers and employees from claims, demands and damages (direct and consequential) of every kind and nature, known and unknown arising out of or in any way connected with such disputes. If You are a California resident or an entity incorporated in California, You waive the applicability of California Civil Code Section 1542 which says A general release does not extend to claims which the creditor does not know or suspect to exist in such creditor’s favour at the time of executing the release, which if known by the creditor must have materially affected such creditor’s settlement with the debtor
```

Author: Changed as suggested.
